content/en/shift_left.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Shift Left
+status: Feedback Appreciated
+category: Concept
+---
+
+## What it is
+Left in Shift Left refers to the left-to-right (development-to-development) workflow. Shift Left is the practice of implementing tests, security, or other development practices early in the software development cycle rather than towards the end. The actual stages in the software development lifecycle may be defined in different ways based on the team and the models used.
+
+Although originally used to refer to the process of testing early, Shift Left can now also be applied to other aspects of software development and [DevOps](https://glossary.cncf.io/devops/), such as security and deployment. 
+
+## Problem it addresses
+Security issues, bugs, and software defects can be more difficult and expensive to fix if they are discovered late in the development cycle or after deployment, particularly if the software has already been deployed into production. 
+
+## How it helps
+By adopting a shift left mindset for software development, teams can implement testing and security throughout the development lifecycle. And because responsibility for testing and security is shared across the development team — from software engineers to quality assurance to DevOps/operations — everyone plays a part in ensuring the stability and security of an application. 
```

Comment:
Uppercased Shift Left for consistency and removed DevOps personification. 

```suggestion
By adopting a Shift Left mindset for software development, teams can implement testing and security throughout the development lifecycle. And because responsibility for testing and security is shared across the development team — from software engineers to quality assurance to operations — everyone plays a part in ensuring the stability and security of an application. 
```